duckduckgo.safariextension/html/global.html

```diff
@@ -117,6 +117,15 @@
     safari.application.addEventListener("command", ddgRightClickCommand, false);
     safari.application.addEventListener("contextmenu", ddgContextMenu, false);
 
+    // use previously used localStorage['last_version'] to determine whether
+    // this is a new install or not
+    // https://github.com/duckduckgo/safari-zeroclickinfo/commit/4746f30668033551025847a33234ad15579f53d5#diff-89d925e2bf34fb6e3d713637cfd75bdbL39
+    if (typeof localStorage['last_version'] == "undefined") {
+      if (typeof localStorage['atb'] == "undefined") {
+        localStorage['atb'] = 'v' + Math.ceil((new Date().getTime() - 1456333200000) / 604800000);
+      }
+    }
```

Comment:
@bsstoner let me see if I can help.

The `localStorage['atb']` is actually always contained in this `global.html` page -- it never goes out. The addition of the ATB param also happens in here, on line 148 and below. The page you linked also says

> When used from a global HTML page or extension bar, the domain of the local storage is the extension. This data belongs to your extension.

I believe this is the case here.

Did that help?